go/ct/spc/specification.go

```diff
@@ -719,6 +719,129 @@ func getAllRules() []Rule {
 		effect:    NoEffect().Apply,
 	})...)
 
+	// --- TLOAD ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        TLOAD,
+		staticGas: 100,
+		pops:      1,
+		pushes:    1,
+		parameters: []Parameter{
+			StorageAccessKeyParameter{},
+		},
+		conditions: []Condition{
+			RevisionBounds(R13_Cancun, NewestSupportedRevision),
+			IsTransientSet(Param(0)),
```

Comment:
`IsTransientSet` is not a good name. If somebody knows the data model of the EVM, it is not clear what this should mean. Any ideas for a better name?

Reply:
`HasTransientStorageValue(Param(0))` ?